### ISSUE TITLE
when `classes==2`, `"AssertionError: Two classes is binary classification.          Just specify the posative class value"`

### ISSUE THREAD

**Issue by sgbaird** [nasa/pretrained-microscopy-models#4]
```python
import pretrained_microscopy_models as pmm
# setup a UNet model with a ResNet50 backbone.
model = pmm.segmentation_training.create_segmentation_model('Unet', 'resnet50', 'micronet', classes=2)
```

> ```python
> ---------------------------------------------------------------------------
> AssertionError                            Traceback (most recent call last)
> [<ipython-input-18-e7bebb562564>](https://localhost:8080/#) in <module>()
>       2 
>       3 # setup a UNet model with a ResNet50 backbone.
> ----> 4 model = pmm.segmentation_training.create_segmentation_model('Unet', 'resnet50', 'micronet', classes=2)
> 
> [/usr/local/lib/python3.7/dist-packages/pretrained_microscopy_models/segmentation_training.py](https://localhost:8080/#) in create_segmentation_model(architecture, encoder, encoder_weights, classes, activation)
>      75     # setup and check parameters
>      76     assert classes != 2, "Two classes is binary classification.  \
> ---> 77         Just specify the posative class value"
>      78 
>      79     if activation is None:
> 
> AssertionError: Two classes is binary classification.          Just specify the posative class value
> ```

I'm not sure what is meant by "just specify the [positive] class value". Is the suggestion to use a different function?

reproducer: https://colab.research.google.com/drive/1eGvA7_q1dgIEZjHJvugZ0aa-koY3Vg_c#revisionId=0B0MK2L13BxuWaktYK3BYT3RscWxuVXBUQkZIbHNia2M4eCs0PQ

**Comment by JStuckner** [comment]
If you are doing binary classification then just set classes=1 and the background class is implicit. E.g., "grain boundary" and "not grain boundary", but the "not grain boundary" is implicit. In this case the model uses a sigmoid activation with 1 output neuron. If you have 2 positive classes and a background (e.g., grain boundary, porosity, everything else) then set classes=3 and the model will use a softmax activation with 3 output neurons.

Basically, there is never any reason for the model to have 2 output neurons.

By the way, how are you labeling the training data?

**Comment by sgbaird** [comment]
Ok, good to know. Might be worth modifying the error to say "If you are doing binary classification then set classes=1 and the background class is implicit" or similar.

> By the way, how are you labeling the training data?

I haven't run an example with training data yet. Would probably use Fiji, Gimp, or similar, and label the background as black and grain boundaries as white or vice-versa.

I gave it a shot with `classes=1` but was running into a different error #5. Not specific to `classes=1` though, same error for `classes=3` which was working before (not exactly sure what changed).

**Comment by JStuckner** [comment]
Good idea!
